e2etests/testcase_test.go

```diff
@@ -3972,4 +3972,48 @@ All OpenShift Container Platform 4.10 users are advised to upgrade to these upda
 			},
 		},
 	},
+	{
+		image:                   "quay.io/rhacs-eng/qa:drools-CVE-2021-41411",
+		registry:                "https://quay.io",
+		username:                os.Getenv("QUAY_RHACS_ENG_RO_USERNAME"),
+		password:                os.Getenv("QUAY_RHACS_ENG_RO_PASSWORD"),
+		source:                  "NVD",
+		namespace:               "rhel:8",
+		onlyCheckSpecifiedVulns: true,
+		expectedFeatures: []apiV1.Feature{
+			{
+				Name:          "drools",
+				VersionFormat: component.JavaSourceType.String(),
+				Version:       "6.4.0.final",
+				Location:      "org.drools.drools-core-6.4.0.Final.jar:drools-core",
+				Vulnerabilities: []apiV1.Vulnerability{
+					{
+						Name:        "CVE-2021-41411",
+						Description: "drools <=7.59.x is affected by an XML External Entity (XXE) vulnerability in KieModuleMarshaller.java. The Validator class is not used correctly, resulting in the XXE injection vulnerability.",
+						Link:        "https://nvd.nist.gov/vuln/detail/CVE-2021-41411",
+						Severity:    "Critical",
+						FixedBy:     "7.60.0.Final",
```

Comment:
I saw in NVD that `FixedBy` excludes `7.6.0`. Is this correct?

![image](https://github.com/stackrox/scanner/assets/291493/ab42c107-6172-40ee-a65f-d546d5fb02ef)

Reply:
`Up to (excluding)` I believe is meant to mean all versions prior to `7.6.0` but not including `7.6.0` itself.  The NVD data appears bad, the description in NVD contradicts the highlighted section and starts with `drools <=7.59.x is affected by ...` (notice y version `59` in double digits as opposed to the `6` in the CPE block).  The [GitHub advisory data](https://github.com/advisories/GHSA-rc57-9r3x-98cq) appears accurate:

![image](https://github.com/stackrox/scanner/assets/119438707/94a9d2a6-f0a6-4835-bd3c-ffa3bd4f8aef)